frontend/src/container/FormAlertRules/ChartPreview/styles.ts

```diff
@@ -10,7 +10,8 @@ export const NotFoundContainer = styled.div`
 
 export const FailedMessageContainer = styled(Tooltip)`
 	position: absolute;
-	top: 10px;
+	color: #f14;
```

Comment:
can we make use of colors from `antd/colors` ?

Reply:
done